### DISCUSSION TITLE
Other blockers to a 3.0.1 release?

### DISCUSSION THREAD

**Discussion by djmitche** [GothenburgBitFactory/taskwarrior#3374] [General]
Please link to any un-solved issues that should block a 3.0.1 release. 

https://github.com/orgs/GothenburgBitFactory/projects/4/views/1?filterQuery=iteration%3A%223.0+Fixes%22 contains the bugs addressed so far.

**Comment by ryneeverett** [comment]
I think all the low-hanging fruit of the 3.0.0 release has been plucked.

**Reply by djmitche** [reply]
Thanks! I'll probably get time later this week or at least this weekend to make the release (but as always, no promises!)
